altair/vegalite/v5/schema/_config.py

```diff
@@ -0,0 +1,1055 @@
+    binSpacing: float
+    blend: Blend_T
+    color: str | LinearGradient | RadialGradient | ColorName_T
+    continuousBandSize: float
+    cornerRadius: float
+    cornerRadiusBottomLeft: float
+    cornerRadiusBottomRight: float
+    cornerRadiusTopLeft: float
+    cornerRadiusTopRight: float
+    cursor: Cursor_T
+    description: str
+    dir: TextDirection_T
+    discreteBandSize: float | RelativeBandSize
+    dx: float
+    dy: float
+    ellipsis: str
+    endAngle: float
+    fill: str | None | LinearGradient | RadialGradient | ColorName_T
+    fillOpacity: float
+    filled: bool
+    font: str
+    fontSize: float
+    fontStyle: str | FontStyle
+    fontWeight: FontWeight_T
+    height: float
+    href: str
+    innerRadius: float
+    interpolate: Interpolate_T
+    invalid: None | MarkInvalidDataMode_T
+    limit: float
+    lineBreak: str
+    lineHeight: float
+    minBandSize: float
+    opacity: float
+    order: bool | None
+    orient: Orientation_T
+    outerRadius: float
+    padAngle: float
+    radius: float
+    radius2: float
+    shape: str | SymbolShape
+    size: float
+    smooth: bool
+    startAngle: float
+    stroke: str | None | LinearGradient | RadialGradient | ColorName_T
+    strokeCap: StrokeCap_T
+    strokeDash: Sequence[float]
+    strokeDashOffset: float
+    strokeJoin: StrokeJoin_T
+    strokeMiterLimit: float
+    strokeOffset: float
+    strokeOpacity: float
+    strokeWidth: float
+    tension: float
+    text: str | Sequence[str]
+    theta: float
+    theta2: float
+    timeUnitBandPosition: float
+    timeUnitBandSize: float
+    tooltip: str | bool | None | float | TooltipContent
+    url: str
+    width: float
+    x: str | float
+    x2: str | float
+    y: str | float
+    y2: str | float
+
+
+class AreaConfigKwds(TypedDict, total=False):
+    """Placeholder doc."""
+
+    align: Align_T
+    angle: float
+    aria: bool
+    ariaRole: str
+    ariaRoleDescription: str
+    aspect: bool
+    baseline: str | Baseline_T
+    blend: Blend_T
+    color: str | LinearGradient | RadialGradient | ColorName_T
+    cornerRadius: float
+    cornerRadiusBottomLeft: float
+    cornerRadiusBottomRight: float
+    cornerRadiusTopLeft: float
+    cornerRadiusTopRight: float
+    cursor: Cursor_T
+    description: str
+    dir: TextDirection_T
+    dx: float
+    dy: float
+    ellipsis: str
+    endAngle: float
+    fill: str | None | LinearGradient | RadialGradient | ColorName_T
+    fillOpacity: float
+    filled: bool
+    font: str
+    fontSize: float
+    fontStyle: str | FontStyle
+    fontWeight: FontWeight_T
+    height: float
+    href: str
+    innerRadius: float
+    interpolate: Interpolate_T
+    invalid: None | MarkInvalidDataMode_T
+    limit: float
+    line: bool | OverlayMarkDef
+    lineBreak: str
+    lineHeight: float
+    opacity: float
+    order: bool | None
+    orient: Orientation_T
+    outerRadius: float
+    padAngle: float
+    point: str | bool | OverlayMarkDef
```

Comment:
TODO: `OverlayMarkDef`

Reply:
Fixed in [`da516c1` (#3536)](https://github.com/vega/altair/pull/3536/commits/da516c1a6753b1fc3520d6d9b0dc689a2b22de00)